contracts/contracts/ChatOracle.sol

```diff
@@ -25,6 +71,12 @@ interface IChatGpt {
         uint callbackId
     ) external view returns (string[] memory);
 
+
+    function onOpenAiOracleLlmResponse(
```

Comment:
Would probably better be called `onOracleOpenAiLlmResponse` to reflect that it is our oracle, not openai's oracle.